lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -167,6 +167,14 @@ class BasicEntityPersister implements EntityPersister
      */
     private $insertSql;
 
+    /**
+     * List of columns of the currently cached insertSql.
+     * The insertSql is reset if another column list is given which happens e.g. when changing the id generator.
+     *
+     * @var array
```

Comment:
It this an array of column names, if yes, it should probably be
```suggestion
     * @var string[]`
```